### ISSUE TITLE
No such file or directory when installing module

### ISSUE THREAD

**Issue by konrness** [kayakco/puppet-r9util#1]
```
root@BambooAgentPHP:/etc/puppet/modules# puppet module install kayak-r9util
Preparing to install into /etc/puppet/modules ...
Downloading from http://forge.puppetlabs.com ...
Installing -- do not interrupt ...
Error: No such file or directory - /etc/puppet/modules/r9util/spec/fixtures/modules/r9util
Error: Try 'puppet help module install' for usage
```

This issue occurs when installing the module. The /etc/puppet/modules/r9util/spec/fixtures directory does not exist in the code-base.

Ubuntu 12.04 LTS
Puppet 2.7.23
Modules installed:

```
root@BambooAgentPHP:/etc/puppet/modules# puppet module list
Warning: Missing dependency 'kayak-r9util':
  'kayak-bamboo_agent' (v0.0.7) requires 'kayak-r9util' (>= 0.0.3)
/etc/puppet/modules
âââ kayak-bamboo_agent (v0.0.7)
âââ puppetlabs-git (v0.0.3)
âââ puppetlabs-java (v1.1.0)
âââ puppetlabs-stdlib (v4.1.0)
âââ r9util (???)
```

**Comment by chqr** [comment]
I suspect this is happening because spec/fixtures/modules/r9util is a symlink in the 0.0.3 release, and Puppet 2.7 doesn't like it. I have released a new version of the r9util module, 0.0.4, which does not include the fixtures directory. Could you try installing again?

**Comment by konrness** [comment]
Too late. I tried upgrading to Puppet 3.4.3 and it works now. Closing.
